spec/lib/declare_schema/migration_generator_spec.rb

```diff
@@ -1093,4 +1093,29 @@ class Advert < active_record_base_class.constantize
       expect(base_class).to eq("(Rails::VERSION::MAJOR >= 5 ? ActiveRecord::Migration[4.2] : ActiveRecord::Migration)")
     end
   end
```

Comment:
We always want the test in the matching file. We're still adapting this hobo code to be like that. Can you create a field_spec_spec and show that `different_to?` returns false in this case?